.github/copilot-instructions.md

```diff
@@ -0,0 +1,126 @@
+- No unit tests exist in this repository
+- WARNING: `npm run lint` has pre-existing errors in hot-potato module and eslint config deprecation - these are not your responsibility to fix unless working on those specific files
+
+### Manual Application Testing
+- The bot cannot be fully tested without valid Discord credentials
+- With valid credentials, test the Discord slash commands:
+  - `/ping` - Basic connectivity test
+  - `/bonjour` - Simple greeting command
+  - `/sutom start` - Start SUTOM word game
+  - `/roulette-russe` - Russian roulette game
+  - `/planifier` - Event planning modal
+- Application startup should show "Isabelle is ready to serve! 🚀" message
+
+### Code Quality Checks
+- Husky pre-commit hooks run lint-staged (ESLint + Prettier) automatically
+- ESLint configuration is strict TypeScript with recommended rules but has pre-existing errors
+- Prettier enforces single quotes, trailing commas, 2-space tabs, semicolons
+- Focus on fixing lint errors only for code you modify, not pre-existing issues
+
+## Build System and Timing
+
+### Command Timing Expectations (measured on standard CI environment)
+- `npm install`: 15-40 seconds (network dependent, requires Node.js >=22 but works on 20 with warnings)
+- `npm run build`: <1 second (tsup is extremely fast)
+- `npm run lint`: ~5-6 seconds (but has pre-existing errors - ignore unless working on affected files)
+- `npm run prettier`: <1 second
+- `npm run db:migrate`: <1 second
+- `npm run dev`: <2 seconds to start (includes migration + attempts Discord connection)
+
+### Build Artifacts
+- TypeScript compiled to `dist/index.mjs` (ESM module)
+- Source maps generated as `dist/index.mjs.map`
+- Resources copied to `dist/resources/` from `public/resources/`
+
+## Common Tasks
+
+### Repository Structure
+```
+src/
+├── config.ts          # Environment configuration
+├── db/                # Drizzle ORM database setup
+├── index.ts           # Main application entry point
+├── manager/           # Command, interaction, and config managers
+├── modules/           # Discord bot modules (games, utilities)
+└── utils/             # Shared utilities (cache, dates, mentions, etc.)
+
+public/resources/      # Static resources (word lists, etc.)
+drizzle/              # Database migration files
+.github/workflows/    # CI/CD pipelines
+```
+
+### Key Modules
+- `CoreModule`: Basic commands (ping, bonjour)
+- `SutomModule`: French word guessing game with canvas rendering
+- `RussianRouletteModule`: Timeout-based roulette game
+- `PlanifierModule`: Event planning with Discord forum integration
+- `AutomaticResponsesModule`: Configurable auto-responses
+- `ScheduleModule`: Calendar integration via .ics files
+- `HotPotato`: Role-passing game
+
+### Database Operations
+- Uses Drizzle ORM with SQLite via libSQL
+- Schema defined in `src/db/schema.ts`
+- Migrations in `drizzle/` directory
+- Two main tables: `automatic_responses`, `guild_configs`
```

Comment:
This is subject to change and is therefore not about how to build the project, remove that line